app/views/gardens/_area.html.haml

```diff
@@ -0,0 +1,15 @@
+- if garden.area
+  %span.area
+    = "#{garden.area} "
+    - if garden.area_unit === 'square metre'
+      %span{:title => 'square metre'}
+        m
+        %sup 2
+    - elsif garden.area_unit === 'square foot'
+      %span{:title => 'square foot'}
+        ft
+        %sup 2
+    - elsif garden.area_unit === 'hectare'
+      %span{:title => 'hectare'} HA
+    - elsif garden.area_unit === 'acre'
+      %span{:title => 'acre'} A
```

Comment:
Hi @tygriffin I like it too :P
Just some questions..
Where should I put the code? Should i generate a new helper or a variable in the partial is good enough?